src/WebJobs.Script.WebHost/Program.cs

```diff
@@ -79,6 +77,13 @@ public static IWebHostBuilder CreateWebHostBuilder(string[] args = null)
 
                     loggingBuilder.AddDefaultWebJobsFilters();
                     loggingBuilder.AddWebJobsSystem<WebHostSystemLoggerProvider>();
+
+                    if (!FeatureFlags.IsEnabled(ScriptConstants.FeatureFlagDisableDiagnosticEventLogging))
```

Comment:
Unrelated to change in this PR, But I noticed we [call string.split ](https://github.com/Azure/azure-functions-host/blob/28e9550320b13137f4eaf16483ebcdb630699506/src/WebJobs.Script/Config/FeatureFlags.cs#L22-L23) and `Contains` every time we call the `IsEnabled` method. Wonder there is a way for us to cache that to may be a hashset?  Also should we be reading that env variable every time that method is called?

Reply:
I just updated this in my recent commit -- no longer check it every time; instead we cache and only re-evaluate if specialization has occurred (which is really the only way it can change)

Reply:
We could probably come back to this and improve that `FeatureFlags` thing entirely to cache internally. But it is fairly scarcely used, I believe so probably not a huge priority.